packages/reactive-element/src/decorators/query.ts

```diff
@@ -64,6 +82,19 @@ export function query(selector: string, cache?: boolean): QueryDecorator {
     descriptor?: PropertyDescriptor
   ) => {
     const doQuery = (el: Interface<ReactiveElement>): V => {
+      if (DEV_MODE && !el.hasUpdated) {
+        const name =
+          typeof nameOrContext === 'object'
+            ? nameOrContext.name
+            : nameOrContext;
+        issueWarning(
+          'early-query-field-access',
```

Comment:
This will result in the warning linking to `lit.dev/msg/early-query-field-access`, and we don't yet have any information there. Instead could this be an empty string, and `issueWarning` should only add the link if a code is provided.

Copy the code pattern in lit-html: https://github.com/lit/lit/blob/main/packages/lit-html/src/lit-html.ts#L221-L223

Or:

```
    warning += code
      ? ` See https://lit.dev/msg/${code} for more information.`
      : '';
```